src/js/angular/aclmanagement/model.js

```diff
@@ -4,94 +4,155 @@ export class ACListModel {
      * @param {ACRuleModel[]} aclRules
      */
     constructor(aclRules = []) {
-        this._aclRules = aclRules;
+        this._aclRules = new Map();
```

Comment:
The missing types for the key and the value for this Map leads to weird type resolutions in other places where you use it.

Reply:
added map types